pql/parser.go

```diff
@@ -26,8 +26,9 @@ import (
 // timeFormat is the go-style time format used to parse string dates.
 const timeFormat = "2006-01-02T15:04"
 
-// duplicateArgErrorMessage is used as an error string in the parser.
+// error strings in the parser
 const duplicateArgErrorMessage = "duplicate argument provided"
+const outOfBoundsErrorMessage = "out of bounds"
```

Comment:
I think the text of the error could be a bit more descriptive: "integer is not in signed 64-bit range" and then call the error message `intOutOfRangeError` maybe.